src/main/kotlin/com/terraformation/backend/tracking/api/ObservationsController.kt

```diff
@@ -395,12 +395,63 @@ class ObservationsController(
     return SimpleSuccessResponsePayload()
   }
 
+  @GetMapping("/adHoc")
+  @Operation(summary = "Gets a list of observations of planting sites.")
+  fun listAdHocObservations(
+      @RequestParam
+      @Schema(
+          description =
+              "Limit results to observations of planting sites in a specific organization. " +
+                  "Ignored if plantingSiteId is specified.")
+      organizationId: OrganizationId? = null,
+      @RequestParam
+      @Schema(
+          description =
+              "Limit results to observations of a specific planting site. Required if " +
+                  "organizationId is not specified.")
+      plantingSiteId: PlantingSiteId? = null,
+  ): ListAdHocObservationsResponsePayload {
+    val observations: Collection<ExistingObservationModel>
+    val plantingSites: Map<PlantingSiteId, ExistingPlantingSiteModel>
+    val plotCounts: Map<ObservationId, ObservationPlotCounts>
+
+    if (plantingSiteId != null) {
+      observations =
+          observationStore.fetchObservationsByPlantingSite(plantingSiteId, isAdHoc = true)
+      plantingSites =
+          mapOf(
+              plantingSiteId to
+                  plantingSiteStore.fetchSiteById(plantingSiteId, PlantingSiteDepth.Site))
+      plotCounts = observationStore.countPlots(plantingSiteId)
```

Comment:
Maybe this should take an argument to control whether it's counting plots in ad-hoc or assigned observations so we don't waste database time counting plots for observations we'll ignore.